README.md

```diff
@@ -207,11 +207,11 @@ Just specify a GitHub repository in the URL and off you go!
 - [ ] add GitHub search
 
 
-[gh-board-image]: https://img.shields.io/github/issues/philschatz/gh-board.svg?label=Issues%20%28gh-board%29
+[gh-board-image]: https://img.shields.io/github/issues/coala/gh-board.svg?label=Issues%20%28gh-board%29
 [gh-board-url]: http://philschatz.com/gh-board/
```

Comment:
Needs update. Probably https://github.com/coala/gh-board/issues
Also, `gh-board-image` and `gh-board-url` are bad names. Probably `issues-image` and `issues-url`